doc/1.7/dev_guide/lua_style_guide.rst

```diff
@@ -214,7 +221,38 @@ Zen of Python is good; understand it and use wisely:
         local thing = {1, 2, 3}
         -- good
 
-  - Add a line break after multiline blocks.
+  - use space in map definitions around equality sign and commas
+
+    .. code-block:: lua
+
+        return {1,2,3,4} -- bad
+        return {
+            key1 = val1,key2=val2
+        } -- bad
+
+        return {
+            1, 2, 3, 4
+            key1 = val1, key2 = val2,
+            key3 = vallll
+        } -- good
+
+    also, you may use alignment:
+
+    .. code-block:: lua
+
+        return {
+            long_key  = 'vaaaaalue',
+            key       = 'val',
+            something = 'even better'
+        }
+
+  - surround top-level function and class definitions with two blank lines.
```

Comment:
It’s suggested by only one of three code styles you linked above. Personally I don’t like Python’s approach with two lines btw each top-level function/block. Moreover, in Lua we have `end`s after each function (on it’s own line), so two lines seems even more useless then in a Python code. Anyway, it’s point where third opinion would be helpful.

Maybe I confused you in our voice talk, sorry.